exercises/practice/queen-attack/queen-attack.rakutest

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env raku
+use Test;
+use lib $?FILE.IO.dirname;
+use QueenAttack;
+
+try { # begin: 3ac4f735-d36c-44c4-a3e2-316f79704203
+  my $QA = QueenAttack.new( :row(2), :column(2) );
+} // fail "Test creation of Queens with valid and invalid positions: queen with a valid position"; # end: 3ac4f735-d36c-44c4-a3e2-316f79704203
+
+throws-like( # begin: 4e812d5d-b974-4e38-9a6b-8e0492bfa7be
+  { QueenAttack.new( :row(-2), :column(2) ) },
+  Exception,
+  message => /:i
+      "row not positive"
+      || "type check failed"
+  /,
+  "Test creation of Queens with valid and invalid positions: queen must have positive row",
+); # end: 4e812d5d-b974-4e38-9a6b-8e0492bfa7be
+
+throws-like( # begin: f07b7536-b66b-4f08-beb9-4d70d891d5c8
+  { QueenAttack.new( :row(8), :column(4) ) },
+  Exception,
+  message => /:i
+      "row not on board"
+      || "type check failed"
+  /,
+  "Test creation of Queens with valid and invalid positions: queen must have row on board",
+); # end: f07b7536-b66b-4f08-beb9-4d70d891d5c8
+
+throws-like( # begin: 15a10794-36d9-4907-ae6b-e5a0d4c54ebe
+  { QueenAttack.new( :row(2), :column(-2) ) },
+  Exception,
+  message => /:i
+      "column not positive"
+      || "type check failed"
+  /,
+  "Test creation of Queens with valid and invalid positions: queen must have positive column",
+); # end: 15a10794-36d9-4907-ae6b-e5a0d4c54ebe
+
+throws-like( # begin: 6907762d-0e8a-4c38-87fb-12f2f65f0ce4
+  { QueenAttack.new( :row(4), :column(8) ) },
+  Exception,
+  message => /:i
+      "column not on board"
+      || "type check failed"
+  /,
+  "Test creation of Queens with valid and invalid positions: queen must have column on board",
+); # end: 6907762d-0e8a-4c38-87fb-12f2f65f0ce4
+
+cmp-ok( # begin: 33ae4113-d237-42ee-bac1-e1e699c0c007
+  QueenAttack.new( :row(2), :column(4) ).is-attack( :row(6), :column(6) ),
+  &infix:<==>,
+  Bool::False,
+  "Test the ability of one queen to attack another: cannot attack",
+); # end: 33ae4113-d237-42ee-bac1-e1e699c0c007
+
+cmp-ok( # begin: eaa65540-ea7c-4152-8c21-003c7a68c914
+  QueenAttack.new( :row(2), :column(4) ).is-attack( :row(2), :column(6) ),
+  &infix:<==>,
+  Bool::True,
```

Comment:
You don't need to use the `.raku` method on boolean results in the template. You get `True`/`False` without it which I think is sufficient:

https://github.com/exercism/raku/blob/main/exercises/practice/leap/.meta/template-data.yaml